src/main/webapp/app/course/plagiarism-cases/instructor-view/plagiarism-cases-instructor-view.component.html

```diff
@@ -49,20 +56,23 @@ <h5 class="mb-0">
                     </div>
                 </div>
                 <div class="card-body">
-                    @for (plagiarismCase of groupedPlagiarismCases[exercise!.id!]; track plagiarismCase) {
+                    @for (plagiarismCase of groupedPlagiarismCases[exercise!.id!]; track plagiarismCase; let j = $index) {
                         <div class="row mb-3">
                             <div class="col-1 text-center">
                                 <a [routerLink]="['/course-management', courseId, 'plagiarism-cases', plagiarismCase.id]">
-                                    {{ plagiarismCase.student.name }} ({{ plagiarismCase.student.login }})
+                                    {{ plagiarismCase.student?.name }} ({{ plagiarismCase.student?.login }})
```

Comment:
I think, if you already consider the case where this is undefined, you should not display a link when it is.
Maybe wrap this in a @if(..) { ... }